sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/PhpUnitOverallCoverageResultParser.java

```diff
@@ -29,21 +29,19 @@
 public class PhpUnitOverallCoverageResultParser extends PhpUnitCoverageResultParser {
 
   /**
-   * Instantiates a new php unit coverage result parser.
-   *
-   * @param context    the context
+   * Instantiates a new PHPUnit coverage result parser.
    */
-  public PhpUnitOverallCoverageResultParser(SensorContext context, FileSystem fileSystem) {
-    super(context, fileSystem);
+  public PhpUnitOverallCoverageResultParser(FileSystem fileSystem) {
+    super(fileSystem);
     linesToCoverMetric = CoreMetrics.OVERALL_LINES_TO_COVER;
     uncoveredLinesMetric = CoreMetrics.OVERALL_UNCOVERED_LINES;
     coverageLineHitsDataMetric = CoreMetrics.OVERALL_COVERAGE_LINE_HITS_DATA;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "OVERALL_COVERAGE_LINE_HITS_DATA"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)